index.html

```diff
@@ -7,5 +7,7 @@
 </head>
 <body>
     <h2>Dit is een andere titel, die gaat niet over Pim</h2>
```

Comment:
Een kleine suggestie voor de aanpassing van de titel kan zijn:

```suggestion
    <h2>Opdracht voor de workshop Git deel 2</h2>
```